Cargo.toml

```diff
@@ -24,7 +24,7 @@ pyo3 = { version = "0.15", default-features = false }
 pyo3 = { version = "0.15", features = ["auto-initialize"] }
 
 [features]
-# In default setting, python version is automatically detected
```

Comment:
This comment looks very out-of-date, perhaps better to just delete it? I think it's from when the python 2 / python 3 version had to be set.